en_US/profiles.md

```diff
@@ -26,53 +43,114 @@ date: 2016-05-15
     * [Email](mailto:dave@lab6.com)
     * IRC: davelab6
 
+### D
+
+* [Aneesh Dogra](https://wiki.sugarlabs.org/go/User:Aneesh_Dogra){:target="_blank"}  
+    * **E-mail**: &lt;first name&gt; at sugarlabs dot org
+    * **About**: Has maintained level tool, IRC
+
 ### F
 
 * Justin W. Flory
+* [Bert Freudenberg](https://wiki.sugarlabs.org/go/User:Bert){:target="_blank"}  
+    * **E-mail**: bert AT freudenbergs DOT de  
+    * **Freenode IRC nickname**: bertf (#sugar, #olpc-de)
+    * **About**: Bert Freudenberg is a freelancing Software Engineer based in Magdeburg, Germany. From 2006-2008 he was contracted to port the Squeak Etoys learning environment to Sugar. Bert still voluntarily maintains the Sugar version of Etoys.
+* [Tony Forster](https://wiki.sugarlabs.org/go/User:Tonyforster){:target="_blank"}  
+    * **E-mail**: forster AT ozonline DOT com DOT au  
+    * **About**: Interests: Activities which maximise the opportunity for self-directed learning: low floor, wide walls and high ceiling with particular emphasis on the walls and ceiling. My Sugar favourites: Turtleart, Scratch, Etoys. Game Maker still my Windows favourite.
+
+      Currently user testing Turtleart and Sliderule, experimenting with Pippy. Supporting the Timor Leste deployment.
+
 
 ### G
 
-* José Miguel García
+* Jose Miguel Garcia
```

Comment:
Fixed in https://github.com/sugarlabs/www-sugarlabs/commit/ef07c066a9be55715c68ee8064bed7600b172108